integrations.tf

```diff
@@ -0,0 +1,24 @@
+module "integrations" {
+  source = "git::https://github.com/truefoundry/terraform-truefoundry-integrations.git?ref=v0.1.0-rc.2"
+
+  count = var.aws_cluster_integration_enabled ? 1 : 0
+
+  cluster_type      = "aws-eks"
+  cluster_name      = var.cluster_name
+  control_plane_url = var.control_plane_url
+
+  tfy_api_key         = var.tfy_api_key
+  trigger_helm_update = var.trigger_helm_update
+
+  aws_region                      = var.aws_region
+  aws_role_arn                    = var.control_plane_roles
+  aws_account_id                  = var.aws_account_id
+  aws_s3_enabled                  = var.feature_blob_storage_enabled
```

Comment:
We are using `var.platform_feature_enabled ? var.feature_blob_storage_enabled ? 1 : 0 : 0` everywhere to check if s3 is to be enabled or not. Should we use `platform_feature_enabled` here as well? cc @dunefro